src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -62,6 +64,18 @@ public with sharing class CRLP_RollupUI_SVC {
         @AuraEnabled public Integer countFilterRules;
         @AuraEnabled public Integer countRollups;
         @AuraEnabled public String name;
+        @AuraEnabled public Id id;
+    }
+
+    /*******************************************************************************************************
+    * @description a class to hold only the required information from the filter_rule__mdt records for the filter group detail.
+    */
+    public class FilterRulesTableItem{
+        @AuraEnabled public String objectLabel;
+        @AuraEnabled public String fieldLabel;
+        @AuraEnabled public String operator;
+        @AuraEnabled public String constant;
+        @AuraEnabled public Id id;
```

Comment:
We included it for when we would need to update a record. Do we not need it here?

Reply:
Not 100% sure yet. The `Id` is not used for updates so it may not be required at all. What is needed is the `DeveloperName` (aka `RecordName`) value.

Reply:
Makes sense. Feel free to add the DeveloperName if you end up working with the filter group save/edit before I get back to it!

Reply:
Would you be opposed to me refactoring the logic that sets the `activeRollupId` attribute to use `activeRollupRecordName` instead (dropping the use of the `Id`)?

Reply:
That would be fine by me! Just be sure to check the use of activeRollupId within the parent RollupsContainer too, because there are a few different places we send in that reference.